Orange-OpenSource/Orange-Boosted-Bootstrap
# 🚀 Release 5.1.2

* Haven't taken into account https://github.com/twbs/bootstrap/commit/78e9b402df84042920d9f2a61e86fc3bb0b4dcea because we haven't exactly the same configuration to handle Dependabot. Maybe we should create an issue to see if our approach is the good one.
  * Shouldn't we use `branches-ignore` too?
  * Bundlewatch is run in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/827 for example while there are `!dependabot/**` rules 

---

  - Firefox ESR
  - IE11 (v4 only)
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
  - then, if bumping a minor or major version:
    - [ ] Ensure Algolia indexes new release content ([probably requires a PR](https://github.com/algolia/docsearch-configs/pull/1954))
  - check wrong matches in `CHANGELOG.md`, and maybe `site/content/docs/<version>/migration.md`
  - :warning: check the `package-lock.json` and `package.json` content, only "boosted" should have its version changed!
  - manually update the version in `nuget/boosted.nuspec` and `nuget/boosted.sass.nuspec`
  - :warning: `site/content/docs/5.1/**/*.md` should not always be modified
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
  - run `conventional-changelog -p angular -i CHANGELOG.md -s`
  - and probably maintain [a ship list (eg for v4.4.0)](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/226)
  - attach zip file
  - paste CHANGELOG / Ship list in the release's description
- [ ] run `build/ship.sh x.x.x` to… (v4 only)
  - Pull the latest changes update new version numbers
  - Compile CSS and JS
  - Compress dist files
  - Generate the hashes and update our config's cdn section
  - Compile the docs
  - Copy-pasta built docs to `bs-docs`-named clone of Boosted.
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
  - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
  - [ ] copy `../_site` (main) or `../bs-docs` (v4-dev) to the `gh-pages` branch
